examples/match.du

```diff
@@ -0,0 +1,45 @@
+match(4){
```

Comment:
Appreciate the example, would be awesome if we could work together on getting some test cases written along with some documentation!